docs/Symbols.md

```diff
@@ -0,0 +1,98 @@
+# Symbols
+
+## Symbols filter settings
+
+By default Outline (symbols) display `DOM elements`, `processing instruction` and `DTD element, entity, attribute list declarations`.
+
+The DOM attributes and text nodes are not displayed in the outline. Indeed to prevent from good performance, symbols must be limitated; displaying attributes and text nodes in the outline could generate a lot of symbols.
+
+Here a sample Outline with maven `pom.xml`:
+
+![Outline of pom.xml without filter](images/Symbols/SymbolsPOMWithoutFilter.png)
+
+In maven context, the text nodes information are important and we should prefer showing text nodes in the Outline:
+
+![Outline of pom.xml with text filter](images/Symbols/SymbolsPOMWithTextFilter.png)
+
+In Spring context, the @Id attribute information are important and we should prefer showing those attributes in the Outline:
+
+![Outline of Spring beans with @id filter](images/Symbols/SymbolsBeansWithIdAttrFilter.png)
+
+In other words, displaying attributes or text nodes depends on the XML file.
+
+`Symbols filters` gives the capability to define filter (to include or exclude) some attributes, some text nodes for a given XML file kind.

```

Comment:
This should either be 'Symbol filters' without the back ticks, or `xml.symbols.filters` . Usually things in back ticks indicate they are a key word but in this case "Symbol filters" is just the formal name for the feature.

Reply:
I agree, I replaced with  `xml.symbols.filters`